etc/ReplayOfflineConfiguration.py

```diff
@@ -244,15 +243,15 @@
                  timePerEvent=4,
                  sizePerEvent=1700,
                  versionOverride=expressVersionOverride)
- 
+
 addExpressConfig(tier0Config, "ExpressCosmics",
                  scenario=cosmicsScenario,
                  diskNode="T0_CH_CERN_Disk",
                  data_tiers=["FEVT"],
                  write_dqm=True,
                  alca_producers=["SiStripPCLHistos", "SiStripCalZeroBias", "TkAlCosmics0T",
                                  "SiPixelCalZeroBias",
-                                 "PromptCalibProdSiStrip", "PromptCalibProdSiPixel", "SiStripCalCosmics"
```

Comment:
why is this removed?
Also shouldn't there be `SiPixelCalCosmics` as well?

Reply:
- I'm not sure why `"PromptCalibProdSiStrip", "PromptCalibProdSiPixel"` are removed.
- `SiStripCalCosmics,SiPixelCalCosmics` are assigned to Cosmics PD in the prompt configuration, as was done in CRUZET.

Reply:
`SiPixelCalCosmics` was removed from [AlCa Matrix](https://twiki.cern.ch/twiki/bin/view/CMS/AlCaRecoMatrix#TWFKC_The_Week_Formerly_Known_as) for ExpressCosmics. It is still included in the Cosmics dataset, though. @francescobrivio was this intentional  or is it a mistake?

Reply:
As I said in my previous comment we have requested the same configuration that was in the twiki for CRUZET2021 (https://twiki.cern.ch/twiki/bin/viewauth/CMS/AlCaRecoMatrix#Cosmic_RUn_at_ZEro_Tesla_CRU_AN1). 
Should we update the table to include them for CRAFT?

Reply:
If they're in Prompt, that's fine.

Reply:
> so @germanfgv please just add back `"PromptCalibProdSiStrip", "PromptCalibProdSiPixel"`! thanks!
> 
> (I will review the full configuration asap!)

@francescobrivio those producers are already there in the ExpressCosmics configuration. They have not been removed. Are you referencing some other Stream?